docs/bigquery/snippets.py

```diff
@@ -2161,8 +2163,10 @@ def test_undelete_table(client, to_delete):
     # time is at least within the creation lifespan of the table.  Trying
     # to snapshot a table prior to its creation time is an error.
     table = client.get_table(table_ref)
```

Comment:
In "real life" how would the user get the creation time if they accidentally deleted the table?

Reply:
Real world use typically leverages awareness of the deletion event time.  The bounds checking here is really only necessary because the lifecycle of this table is measured in milliseconds, which is short enough to require the extra care.